fera_full_external.py

```
@@ -0,0 +1,222 @@
+                global_settings.initiate_processes()
+                start_time = time.time()
+                fera.start_fera_app() 
+            elif(".db" == extension.lower()):
+                global_settings.pathdb = Path(sys.argv[1])    
+                gotoviewer = False
+                if(len(sys.argv) >= 3 and sys.argv[2]=='1'):
+                    gotoviewer = True  
+                #print('waiting3')
+                global_settings.splash_window.window.attributes("-alpha", 255)
+                try:
+                    None
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)